src/moin/storage/middleware/indexing.py

```diff
@@ -410,20 +420,6 @@ def __init__(self, index_storage, backend, wiki_name=None, acl_rights_contents=[
         latest_revisions_schema = Schema(**latest_revs_fields)
         all_revisions_schema = Schema(**all_revs_fields)
 
-        # Define dynamic fields
-        dynamic_fields = [("*_id", ID(stored=True)),
-                          ("*_text", TEXT(stored=True)),
-                          ("*_keyword", KEYWORD(stored=True)),
-                          ("*_numeric", NUMERIC(stored=True)),
-                          ("*_datetime", DATETIME(stored=True)),
-                          ("*_boolean", BOOLEAN(stored=True)),
-                         ]
-
-        # Adding dynamic fields to schemas
-        for glob, field_type in dynamic_fields:
-            latest_revisions_schema.add(glob, field_type, glob=True)
-            all_revisions_schema.add(glob, field_type, glob=True)
-
```

Comment:
OK, will change this back as well, but I have no idea how a user could add their own metadata and get it back in a useful way.

If you are thinking of the Extra Metadata (JSON) display on the item modify view, that was removed 6 months ago: #700. There is also #280 that should be closed (with no additional code changes).

Reply:
Well, the original idea was that users can enter own metadata via extra metadata and that this is indexed and can be quickly searched for. If it helps we can initially leave that away, but in general i think it can be a useful feature.